web/beacon-app/src/features/projects/routes/ProjectDetailPage.tsx

```diff
@@ -32,10 +32,18 @@ const ProjectDetailPage = () => {
 
   useEffect(() => {
     if (!param || !projectID) {
-      navigate(PATH_DASHBOARD.HOME);
+      navigate(PATH_DASHBOARD.PROJECTS);
     }
   }, [param, navigate, projectID]);
 
+  // redirect user to project page if project is not found
+
+  useEffect(() => {
+    if (!hasProject) {
+      navigate(PATH_DASHBOARD.PROJECTS);
+    }
+  }, [hasProject, navigate]);
+
```

Comment:
This fixes the error that occurs when the user switch organization

Reply:
Would it be a little less confusing if the user is directed to the home page of their other organization when this happens? It may not be really obvious that they are on the project page of the other organization, especially since we have the name of the org in one place and it's on the left side.

Reply:
When a user is on the project detail page and wants to switch to another organization, the best approach is to redirect them to the project list page if the project they are currently viewing does not belong to their organization. This ensures a smoother experience by presenting them with a relevant list of projects within their organization. Redirecting them to the home page may not be as useful in this context since they are already engaged with a specific project and would benefit more from exploring related projects within their organization.